Libraries/LibWeb/Painting/Command.h

```diff
@@ -100,6 +100,7 @@ struct DrawRepeatedImmutableBitmap {
 };
 
 struct Save { };
+struct SaveLayer { };
```

Comment:
Ideally, adding the `SaveLayer` command would be in its own commit before this one.